smart_card_connector_app/src/testing_smart_card_simulation.cc

```diff
@@ -309,11 +309,47 @@ std::vector<uint8_t> MakeParametersTransferReply(
   return transfer_reply;
 }
 
+// Builds a RDR_to_PC_NotifySlotChange message.
+std::vector<uint8_t> MakeNotifySlotChangeTransferReply(CcidIccStatus icc_status,
+                                                       bool slot0_changed) {
+  const uint8_t slot0_current_bit = icc_status != CcidIccStatus::kNotPresent;
```

Comment:
For my own understanding: This is 0x00 if `icc_status` is `CcidIccStatus::kNotPresent`, and 0x01 if it is anything else, right?

Reply:
Correct. Let me add a comment to further clarify that.